http/org_service.go

```diff
@@ -241,8 +241,14 @@ func (h *OrgHandler) handleGetOrg(w http.ResponseWriter, r *http.Request) {
 
 // handleGetOrgs is the HTTP handler for the GET /api/v2/orgs route.
 func (h *OrgHandler) handleGetOrgs(w http.ResponseWriter, r *http.Request) {
-	var filter influxdb.OrganizationFilter
 	qp := r.URL.Query()
+
+	var filter influxdb.OrganizationFilter
```

Comment:
done

Reply:
As for `FindUserResourceMappings`,  I would prefer to implement it in https://github.com/influxdata/influxdb/blob/5d63ebb4fc6ed10f869fc9bfab2d9f19078d2210/kv/urm.go#L94

because `opt` do have usages in https://github.com/influxdata/influxdb/blob/5d63ebb4fc6ed10f869fc9bfab2d9f19078d2210/tenant/service_user.go#L148 
and some other places.